.docker/entrypoint/spec/jekyll_config_provider_spec.rb

```diff
@@ -40,6 +40,19 @@
           is_expected.to include('incremental' => true)
         }
       end
+
+      context 'serve returns config' do
+        before(:all) do
+          jekyll_config_provider = Jekyll::PlantUml::JekyllConfigProvider.new(data_dir)
+          @jekyll_config = jekyll_config_provider.provide('serve')
+        end
+
+        subject { @jekyll_config }
+        
+        it {
+          is_expected.to include('livereload_port' => 35729)
```

Comment:
Codacy found an issue: [Use underscores(_) as thousands separator and separate every 3 digits with them.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5780130)